src/utils/google_cloud/google_cloud_compute.py

```diff
@@ -59,8 +59,8 @@ def delete_everything(self) -> None:
                 .list(project=self.gcp_project, region=constants.SERVER_REGION)
                 .execute()["items"]
             )
-        except Exception as e:
-            logger.error(f"Error getting subnets: {e}")
+        except:
+            logger.exception("Error getting subnets:")
```

Comment:
```suggestion
            logger.exception("Error getting subnets")
```

Reply:
I suppose if the stack trace is logged, then you want the colon and should reject this suggestion. 

If you want to remove the colon then it ought to be removed in the other error messages in this PR too.